build_projects/Microsoft.DotNet.Cli.Build.Framework/Command.cs

```diff
@@ -293,37 +293,40 @@ public Command OnErrorLine(Action<string> handler)
             return this;
         }
 
-        private string FormatProcessInfo(ProcessStartInfo info)
+        private string FormatProcessInfo(ProcessStartInfo info, bool includeWorkingDirectory)
```

Comment:
Should we maybe make this an extension method of ProcessStartInfo? It would read better. Also, instead of the boolean param, how about a FormatProcessInfoWithWorkingDirectory that pass this prefix and a FormatProcessInfo that passes the other prefix and then we can have yet another FormatProcessInfo that takes a prefix as a parameter.